src/test/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilderTest.java

```diff
@@ -45,11 +45,10 @@ void beforeEach() {
         tablePrefix = TABLE_PREFIX_PREFIX + counter.incrementAndGet();
     }
 
-    private static List<String> testTables = new ArrayList<>(Arrays.asList("mt_sharedtablestatic_s_s",
-            "mt_sharedtablestatic_s_n", "mt_sharedtablestatic_s_b", "mt_sharedtablestatic_s_nolsi",
-            "mt_sharedtablestatic_s_s_nolsi", "mt_sharedtablestatic_s_n_nolsi",
-            "mt_sharedtablestatic_s_b_nolsi")).stream()
-            .collect(Collectors.toList());
+    private static List<String> testTables = new ArrayList<>(new ArrayList<>(Arrays.asList("mt_sharedtablestatic_s_s",
+        "mt_sharedtablestatic_s_n", "mt_sharedtablestatic_s_b", "mt_sharedtablestatic_s_nolsi",
+        "mt_sharedtablestatic_s_s_nolsi", "mt_sharedtablestatic_s_n_nolsi",
+        "mt_sharedtablestatic_s_b_nolsi")));
```

Comment:
Can be replaced with {} constructor

Reply:
can't both `new ArrayList<>`s be removed (and their corresponding right parentheses)? (also, `ImmutableList.of` is an option instead of `Arrays.asList`.)

Reply:
I swear I had fixed this.  In any case, it's fixed now ... 35ff1a355813d6b5339dde9914d962581c7d0152